mypy.ini

```diff
@@ -12,3 +12,4 @@ disallow_any_unimported = True
 warn_redundant_casts = True
 warn_unused_configs = True
 strict_equality = True
+exclude = test
```

Comment:
Why is it useful to ignore problems in our tests?

Reply:
Oh I thought it was the standard way. Usually tests have all kinds of test doubles (mocks, etc) which often mean the types don't exactly match. If you'd rather have tests 100%  mypy-ok, we can do it I guess.